docs/source/access-data/data-overview.rst

```diff
@@ -4,6 +4,13 @@ Data Overview
 Pub/Sub
 ----------------
```

Comment:
Echoing our zoom conversation: Add description of cost/billing. Maybe include link to GCP prices.

Reply:
✅